spec/github-pages/integration_spec.rb

```diff
@@ -149,6 +149,10 @@ def env
     end
   end
 
+  context "jekyll-commonmark-ghpages" do
+    # noop: this is tested by other things
```

Comment:
Once https://github.com/github/pages-gem/pull/499 merges, we should be able to add a smoke test for this fairly easily. The main thing here to test is interactions between plugins/depdenencies.

Reply:
Aha, thank you for that! Super helpful.